main.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 72px;
+  background: transparent;
+  padding: 0 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 255px;
+  height: 40px;
+  background: white;
+}
+
+nav li {
+  padding: 10px 10px;
+  list-style: none;
+}
+
+nav a {
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-weight: 600;
+  text-decoration: none;
+  color: #344563;
+  line-height: 20px;
+  padding: 10px;
+  border-radius: 4px;
+}
+
+nav a:hover {
+  background: #ff6163;
+  color: #fff;
+}
+
+main {
+  margin-top: 72px;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 756px;
+  width: 920px;
```

Comment:
- [ ] Kindly avoid using static widths on your project, please see the screenshot below, currently, some elements are not aligned properly, there is an horizontal scroll, please fix this 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 3-main-mobile](https://user-images.githubusercontent.com/69638013/185565157-d9282f12-0f80-4381-a37b-0f108ca5fbe2.png)  |  ![Screenshot_391](https://user-images.githubusercontent.com/69638013/185565335-6a535858-521d-4596-880f-1a95245bc718.png)